dora/core/server/master/src/main/java/alluxio/master/job/DoraLoadJob.java

```diff
@@ -274,7 +267,23 @@ private List<LoadSubTask> createSubTasks(UfsStatus ufsStatus) {
     else {
       subTasks.add(new LoadDataSubTask(ufsStatus, mVirtualBlockSize, 0, contentLength));
     }
-    return subTasks;
+    List<LoadSubTask> subTasksWithWorker = assignSubtasksToWorkers(subTasks, workers, mNumReplica);
```

Comment:
so replica also determines num of replica of metadata right?

Reply:
yes